### PR TITLE
support the loop option in html video output

### PR DESCRIPTION
**Pull request by aplavin**
# Description

The `loop` option was only supported for gif output, but is also very convenient in HTML. Adding its support was quite straightforward!

Not sure about testing though: seems like this HTML output is not tested at all for now.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] Added an entry in CHANGELOG.md (for new features and breaking changes)
- [x] Added or changed relevant sections in the documentation
- [ ] Added unit tests for new algorithms, conversion methods, etc.
- [ ] Added reference image tests for new plotting functions, recipes, visual options, etc.

### CODE REVIEW

File: src/ffmpeg-util.jl [hunk 1]
```diff
@@ -63,15 +63,12 @@ struct VideoStreamOptions
...
-        if format == "gif"
-            (loop === nothing) && (loop = 0)
-        end
+        (loop === nothing) && (loop = 0)
 
         # items are name, value, allowed_formats
         allowed_kwargs = [("compression", compression, ("mp4", "webm")),
                           ("profile", profile, ("mp4",)),
-                          ("pixel_format", pixel_format, ("mp4",)),
-                          ("loop", loop, ("gif",))]
```
Comment (asinghvi17) [inline src/ffmpeg-util.jl:74]: I think that `loop` is given to the rendered GIF here, not the HTML.  Can we add this back in?

Comment (asinghvi17) [reply src/ffmpeg-util.jl:74]: Makes sense - thanks!

**Comment by asinghvi17** [comment]
Yeah, I don't think the HTML output is tested either.  That being said, if you put this up in the documentation somehow I can at least move it into a repo branch to make sure it works!

**Comment by asinghvi17** [comment]
`record` just returns a String so I don't think that will work unfortunately

**Comment by SimonDanisch** [comment]
Yeah we should document Record better, it was introduced exactly for this use case.

File: docs/explanations/animation.md [hunk 2]
```diff
@@ -32,11 +32,8 @@ record(fig, "color_animation.mp4", hue_iterator;
         framerate = framerate) do hue
     lineplot.color = HSV(hue, 1, 0.75)
 end
-nothing # hide
```
Comment (SimonDanisch) [inline docs/explanations/animation.md:35]: This should only work if you changed `record` to `Record`, i think?

**Comment by SimonDanisch** [comment]
Good questions, maybe we can add a test?

**Pull request merged**